### ISSUE TITLE
Add support for showOtherMonths in blankDays

### ISSUE THREAD

**Issue by longprao** [charliekassel/vuejs-datepicker#669]
Similar to jQuery Datepicker's "Dates in other months" demo, please add a "showOtherMonths" options to view the day of the previous and next month on the blank cells of the calendar. 

Possibly able to have events on click on those days.

Thanks a lot!

**Comment by mlturner88** [comment]
I would agree that this would be a nice feature to add. If you have a preferred direction I may find the time to submit a PR for this.

**Comment by controversial** [comment]
I'd also appreciate this functionality

**Comment by controversial** [comment]
This was also requested in #317, which was closed after the author forked this repo and [added the functionality](https://github.com/mlgdev/vuejs-datepicker/pull/2/files) to her fork
